baggage/HTTP_HEADER_FORMAT.md

```diff
@@ -101,3 +101,18 @@ For example, if you have non-production requests that flow through the same serv
 ```
 baggage: isProduction=false
 ```
+# Mutating baggage
+A system receiving a `baggege` request header SHOULD send it to outgoing requests.
+A system MAY mutate the value of this header before passing it on.
+
+As the producer and consumer of `baggage` SHOULD be within the same logical system and the content of they
```

Comment:
is this accurate? If some SaaS is used to process data and return it back, and we want the baggage to be preserved, is it considered the same logical system?

Reply:
Yes, I would say so. The SaaS is still part of the application at large. The definition of where the trust boundaries are is left to the implementor. I think the distinction is that on an application (SaaS or not) that I own, I have access to and can make use of baggage there as well. E.g. if I'd call some blackbox API for currency conversion, there is no point in sending baggage as the receiving system is not controlled by me.